app/common/state/ledgerState.js

```diff
@@ -67,12 +67,25 @@ const ledgerState = {
     return state.setIn(['ledger', 'locations', url, prop], value)
   },
 
-  getLocationProp: (state, url, prop) => {
+  getVerifiedPublisherLocationProp: (state, url, prop) => {
```

Comment:
let's remove `prop`, because we changed this to specific publisher function, so it's not needed anymore

Reply:
Doen